src/routers/mint/index.ts

```diff
@@ -15,7 +15,7 @@
 );
 mintRouter.post(
   "/getMagicLinkFromAirtable",
-  // validateIdToken,
+  validateIdToken,
```

Comment:
we gotta make sure we are adding rate limiters to our load balancers. I believe it could be done here https://github.com/designitycom/devops/blob/sandbox/k8s/web-services.yaml#L163

Reply:
opened a ticket for it https://github.com/designitycom/devops/issues/8

Reply:
Will work on rate limiter as we discussed once our app will be in full production mode and there will be load on our app